interfaces/database_backup_manager/v0/README.md

```diff
@@ -0,0 +1,121 @@
+# `database_backup_manager`
+
+## Usage
+
+This relation interface describes how a database backup manager charm would interact with a database charm. A database backup manager will enable a juju admin to:
+
+1. backup the state of a related database to a specified storage
+1. restore the state of a related data from a specified backup in storage
+
+Initially, the backup or restore will be triggered manually against the database backup manager via an action. However, eventually, the juju admin will be able to schedule periodic backups on the database backup manager.
+
+## Direction
+
+```mermaid
+flowchart TD
+    Requirer -- job-id, job-type, storage-interface, storage-config, artifact-path --> Provider
```

Comment:
I am not sure what is the correct spelling in British English. we need to follow British English.
maybe
```suggestion
    Requirer -- job-id, job-type, storage-interface, storage-config, artefact-path --> Provider
```

Reply:
addressed in e9efe849fae6727a19321e3a9837043daec08e3e